src/utils.js

```diff
@@ -63,16 +65,86 @@ async function checkForPlaystationDirectRedirect(checkInterval, onSuccess, versi
         onSuccess();
     } else {
         setTimeout(() => {
-            console.log("No stock detected yet. Scanning again...");
-            console.log("Number of tries", numTries);
-            console.log("");
+            console.log("PS Direct: No queue detected yet. Scanning again...");
+            console.log(`Number of tries: ${numTries}\n`);
             numTries++;
 
             checkForPlaystationDirectRedirect(checkInterval, onSuccess, version, browser, numTries);
         }, checkInterval);
     }
 }
 
+/*  Recursively checks walmart mobile API for online and inStore stock status
```

Comment:
Let's go ahead and start building out the new directory structure here.

Move Walmart-related stuff out of the `utils.js` file and into a new file: `src/retailers/walmart/index.js`